substrate/client/consensus/grandpa/src/warp_proof.rs

```diff
@@ -311,13 +312,28 @@ where
 			.ok_or_else(|| "Empty proof".to_string())?;
 		let (next_set_id, next_authorities) =
 			proof.verify(set_id, authorities, &self.hard_forks).map_err(Box::new)?;
+		let proofs = proof
```

Comment:
Super nitpick, maybe better naming.

```suggestion
		let justifications = proof
```